config/opensearch_dashboards.yml

```diff
@@ -290,3 +290,8 @@
 
 # Set the value to true to enable workspace feature
 # workspace.enabled: false
+
+# Set the backend roles in groups or users, whoever has the backend roles or exactly match the user ids defined in this config will be regard as dashboard admin.
+# Dashboard admin will have the access to all the workspaces and objects inside OpenSearch Dashboards.
+# workspace.dashboardAdmin.groups: ["dashboard_admin"]
+# workspace.dashboardAdmin.users: ["dashboard_admin"]
```

Comment:
The naming `workspace.dashboardAdmin` may confuse me, it sounds like the admin for dashboard type only. I come few names like `workspace.dashboardsAdmin`,
`workspace.admin` or `workpsace.superAdmin`

Reply:
workspace.admin may be confusing as `admin of workspace` stands for the the users who have write permission on specific workspaces, vote for superAdmin.

Reply:
If there is no configuration named `workspace.admin`, would `workspace.superAdmin` also be confused?